qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/FormSteps.java

```diff
@@ -53,6 +53,11 @@ public void tearDown() {
 	public void clickOnManageHtmlFormsPage() {
 		manageHtmlFormsPage = homePage.goToAdministration().clickOnManageHtmlForms();
 	}
+
+	@Then("the system loads manage Html Forms Page")
+	public void systemLoadsManageHtmlFormsPage() {
+		assertPage(manageHtmlFormsPage.waitForPage());
```

Comment:
I remember we opted to rather assert texts over assert page,  did I get the point right?

Reply:
Resolved thanks.However i think it depends upon how to use assertPage and AssertTexts